assets/product-form.js

```diff
@@ -16,6 +16,7 @@ if (!customElements.get('product-form')) {
 
       submitButton.setAttribute('disabled', true);
       submitButton.classList.add('loading');
+      this.querySelector('.loading-overlay__spinner').classList.remove('hidden');
```

Comment:
The reason that we dont rely on the presence of `loading` is because this is also used in the cart and filter and we dont have the `loading` class there